### DISCUSSION TITLE
Перестал работать ютуб недавно, использую сафари

### DISCUSSION THREAD

**Discussion by gamanoid10** [xvzc/SpoofDPI#275] [Q&A]
просто резко беспричинно перестала работать. что делать?

**Comment by Gen5551** [comment]
Это уже вопросы к роскомпозору. Мы ничего не сможем сделать.

**Comment by vehametra** [comment]
У нас вернулись к тактике замедления.
С блокировки. Видимо  ложно положительные срабатывания стали проблемой.
Или ещё какие проблемы возникли.
Так что я обратно юзаю SpoofDPI,  ибо чёрная магия подзаебала немножко

**Reply by intelligence67** [reply]
> У нас вернулись к тактике замедления. С блокировки. Видимо ложно положительные срабатывания стали проблемой. Или ещё какие проблемы возникли. Так что я обратно юзаю SpoofDPI, ибо чёрная магия подзаебала немножко

Можешь подсказать какую команду прописываешь для запуска. Я просто запускаю у себя и у меня уже не работает неделе полторы - две.
